docs/guide/site_factory.md

```diff
@@ -0,0 +1,155 @@
+Next, configure Site Factory by adding empty SiteAccess groups (only one empty group is mandatory) in `config/packages/ezplatform.yaml`:
+
+```yaml
+ezplatform:
+    siteaccess:
+        list: [site]
+        groups:
+            site_group: [site]
+            example_site_factory_group_1: []
+            example_site_factory_group_2: []
+            
+    system:
+        example_site_factory_group_1:
+        example_site_factory_group_2:
+```
+
+Uncomment the `'@EzSystems\EzPlatformSiteFactory\SiteAccessMatcher': ~` SiteAccess matcher in `ezplatform.siteaccess.match`.
+ 
+Add `ezdesign` configuration and configure designs for empty SiteAccess groups:
+
+```yaml
+ezplatform:
+
+...
+
+ezdesign:
+    design_list:
+        example_1: [example_1_template]
+        example_2: [example_2_template]
+        
+ezpublish:
+    system:
+        example_site_factory_group_1:
+            design: example_1
+        example_site_factory_group_2:
+            design: example_2
+```
+
+`ezdesign` defines templates for your sites, so remember to add them before continuing.
+
+### Add templates configuration
+
+Add thumbnails and names for your templates in `config/packages/ez_platform_site_factory.yaml`
+It will connect SiteAccesses with the templates.
+
+```yaml
+ezplatform_site_factory:
+    templates:
+        ez_site1:
+            siteaccess_group: example_site_factory_group_1
+            name: example_site_1
+            thumbnail: /path/to/image/example-thumbnail_1.png
+        ez_site2:
+            siteaccess_group: example_site_factory_group_2
+            name: example_site_2
+            thumbnail: /path/to/image/example-thumbnail_2.png
+```
+
+You can check the results of your work in the Back Office by going to the **Site** tab.
+There, you should be able to add a new site and choose a design for it.
+
+### Define domains 
+
+To be able to see your site online you need to define a domain for it.
+
+In `.env` file change line 2 to: `COMPOSE_FILE=doc/docker/base-dev.yml:doc/docker/multihost.yml`
+
+Take a look into the `doc/docker/multihost.yml` file. 
+Here you will define your domains. 
+To add a new domain you must add it in `command:` and under frontend and backend aliases as shown in the example below.
+
+```yaml hl_lines="3 6 11"
+services:
+  web:
+    command: /bin/bash -c "cd /var/www && cp -a doc/nginx/ez_params.d /etc/nginx && bin/vhost.sh --host-name=site.example.com --host-alias='admin.example.com test.example.com' --template-file=doc/nginx/vhost.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
+    networks:
+      frontend:
+        aliases:
+          - site.example.com
+          - admin.example.com
+          - test.example.com
+      backend:
+        aliases:
+          - site.example.com
+          - admin.example.com
+          - test.example.com
+
+```
+
+Next, you must define the domains in `etc/hosts`.
+
+```bash
+0.0.0.0 site.example.com admin.example.com test.example.com www.admin.example.com
+```
```

Comment:
Please make sure that is clear that this is not a production configuration.

Reply:
Added warning at the beginning of the section